tools/Docs/GenerateAboutTopicTxt.ps1

```diff
@@ -0,0 +1,12 @@
+$aboutFileInputFullName = "C:\Azure\azure-powershell\src\Accounts\Accounts\help\About\about_az.md"
+$aboutFileOutputFullName = "C:\Azure\azure-powershell\src\Accounts\Accounts\help\About\about_az.help.txt"
```

Comment:
We should add comment to explain when and how to use this script.
Also absolution path -> relative path